thrift_sasl/__init__.py

```diff
@@ -104,7 +104,7 @@ def open(self):
   def _send_message(self, status, body):
     header = struct.pack(">BI", status, len(body))
     if isinstance(body, str):
-      body = body.encode()
+      body = bytearray(body)
```

Comment:
Not sure this does the right thing for Python 3 unicode. I suggest you use `six.ensure_binary` (you may need to add `six` to requirements, but almost all Python installations nowadays will be pulling in six somewhere due to 2/3 compatibility)

Reply:
Thanks @wesm -- just pushed the amended patch.

Reply:
Oh -- sorry, didn't notice the comment about not needing isinstance(). I'll update in a moment.